ex1-shai/bootstrap.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+#Install docker
+
+sudo yum update -y 
+sudo yum install -y yum-utils device-mapper-persistent-data lvm2
+sudo yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
+echo "Installing docker"
+sudo yum install -y docker-ce docker-ce-cli containerd.io
+sudo systemctl enable docker
+sudo systemctl start docker
+
+sudo systemctl is-active --quiet docker && echo Docker is running
+
+#Install java
+
+echo "Installing java"
+sudo yum install -y java-1.8.0-openjdk
+#cp /home/shai/.bash_profile /home/shai/.bash_profile_bku
+#export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.191.b12-1.el7_6.x86_64/jre/bin/java |sudo tee /home/shai/.bash_profile
+#source /home/shai/.bash_profile
+
+export JAVA_HOME=/usr/bin/java
+sudo cp /etc/environment /etc/environment-bku
+sudo chmod 777 /etc/environment
```

Comment:
This is not a good practice.

Reply:
Take a look at other PRs.

Reply:
I looked.
Was it enough to just install java and not have to configure it in JAVA_HOME?
Initially I tried to do:
cp /home/shai/.bash_profile /home/shai/.bash_profile_bku
export JAVA_HOME = / usr / lib / jvm / java-1.8.0-openjdk-1.8.0.191.b12-1.el7_6.x86_64 / jre / bin / java | sudo tee /home/shai/.bash_profile
source /home/shai/.bash_profile
But it just deleted the bash_profile file and I did not understand why.

Reply:
Consult over Slack with the other cookies

Reply:
Ok.

Reply:
Nothing changed